src/utils.js

```diff
@@ -28,6 +28,10 @@ export function getDisplayName(WrappedComponent: string | Component): string {
 }
 
 export function validateSpacingProps(props: SpacingProps) {
+  if (isProd) {
```

Comment:
There are some additional optimizations we could do for reflex but when I tried it we were getting 0.05kb more so I don't think it's worth it